scripts/fix_syntax_errors.py

```
@@ -0,0 +1,157 @@
+            # Check if this is a comment or import at module level
+            if (stripped.startswith('import ') or
+                stripped.startswith('from ') or
+                stripped.startswith('#') or
+                stripped.startswith('"""') or
+                stripped.startswith("'''")):
+
+                # If we're in a function but this looks like module-level code, fix indentation
+                if in_function and not line.startswith('    '):
+                    # This should be at module level
+                    in_function = False
+                    indent_level = 0
```

Ruff (0.12.2)

Local variable `indent_level` is assigned to but never used

Remove assignment to unused variable `indent_level`

(F841)